stack.yaml

```diff
@@ -9,20 +9,24 @@ packages:
   - haskell
 
 extra-deps:
-# Stable versions available on Hackage
-- morley-prelude-0.5.2
-- morley-1.19.1
-- lorentz-0.15.1
-- morley-client-0.3.1
-- cleveland-0.3.1
-- indigo-0.6.0
+- git: https://gitlab.com/morley-framework/morley.git
+  commit: 1391e20cf42293a58a90d4c4b9f5f4fb2b3f8dce # morley master
+  subdirs:
+    - code/morley-prelude
+    - code/morley
+    - code/lorentz
+    - code/morley-client
+    - code/cleveland
+
+- git: https://gitlab.com/morley-framework/indigo.git
+  commit: f8a4aa256ccfc9e5c3c43c456de71c6ed0af2982 # indigo morley update branch
```

Comment:
Also reminder to update these to the latest Hackage release and to indigo master after [!55](https://gitlab.com/morley-framework/indigo/-/merge_requests/55)